.github/workflows/dependabot.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+updates:
+  - package-ecosystem: github-actions
+    directory: /
+    schedule:
+      interval: weekly
+
+  - package-ecosystem: npm
+    directory: /
+    schedule:
+      interval: weekly
+      time: "23:00"
+      timezone: Europe/London
+    open-pull-requests-limit: 10
```

Comment:
@UlisesGascon
I think you should add end of a line.

Reply:
@NaoyaTatetsu Why?

Reply:
@shivam-sharma7 
Based on [unicode org site](https://www.unicode.org/reports/tr13/tr13-9.html#:~:text=A%20record%20separator%20is%20used%20to%20separate%20records.%20For%20example%2C%20when%20exchanging%20tabular%20data%2C%20a%20common%20format%20is%20to%20tab%2Dseparate%20the%20cells%2C%20and%20use%20a%20CRLF%20at%20the%20end%20of%20a%20line%20of%20cells.%20This%20function%20is%20not%20precisely%20the%20same%20as%20line%20separation%2C%20but%20the%20same%20characters%20are%20often%20used.), and [dependabot/demo repository](https://github.com/dependabot/demo/blob/main/.github/dependabot.yml).

Reply:
it's not necessary

Reply:
But a good practice

Reply:
I am going to mark this resolved. Thanks for the input but let's try and avoid nit picking and bike shedding until we are aligned on if this will even land in the first place.